apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/sql/SqlStore.java

```diff
@@ -86,6 +89,10 @@ private SqlStore(ApolloSqlHelper dbHelper, FieldsAdapter parser) {
     return changedKeys;
   }
 
+  @Override public void clearAll() {
```

Comment:
Yes, technically all methods in this class should not be called directly. Will add in the javadocs. https://github.com/apollographql/apollo-android/issues/397